Gemfile.lock

```diff
@@ -196,6 +199,8 @@ GEM
       sprockets (>= 3.0.0)
     stimulus-rails (1.2.1)
       railties (>= 6.0.0)
+    tailwindcss-rails (2.0.23-arm64-darwin)
```

Comment:
Since we're using tailwind, consider using some of the tailwind styles: I personally prefer style #2:

https://ddnexus.github.io/pagy/docs/extras/tailwind/#style-2

Reply:
![plain](https://user-images.githubusercontent.com/15097447/222587692-f0c0f5ea-903a-49e1-95a1-c805a67abad2.png)